plugin/src/main/java/io/jenkins/plugins/coverage/metrics/gerrit/GerritCoverageController.java

```diff
@@ -0,0 +1,111 @@
+package io.jenkins.plugins.coverage.metrics.gerrit;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import net.sf.json.JSONObject;
+
+import org.kohsuke.stapler.QueryParameter;
+import org.kohsuke.stapler.WebMethod;
+import org.kohsuke.stapler.json.JsonHttpResponse;
+import org.kohsuke.stapler.verb.GET;
+import hudson.Extension;
+import hudson.model.Job;
+import hudson.model.ParametersAction;
+import hudson.model.Run;
+import hudson.model.UnprotectedRootAction;
+
+import io.jenkins.plugins.coverage.metrics.restapi.ModifiedLinesCoverageApi;
+import io.jenkins.plugins.coverage.metrics.steps.CoverageBuildAction;
+import io.jenkins.plugins.util.JenkinsFacade;
+
+@Extension
+public class GerritCoverageController implements UnprotectedRootAction {
```

Comment:
What is this class for? It does not belong to this plugin and should be moved somewhere else.

Reply:
This is required for using the coverage API in Gerrit since there we have no information about the Jenkins build number that is required for accessing the API. 
We thought some other people could use it aswell, maybe on a different location where others could also add similar extension for other tools for which the API does not work out of the box? 

If this does not belong to this plugin in your opinion, we can only use it in our fork of course. :)

Reply:
Yes, I would prefer to get that file removed. It would make sense that you put it in a small additional plugin that contains the Gerrit and Jenkins bridge.